docs/reference/shutdown/apis/shutdown-api.asciidoc

```diff
@@ -1,17 +1,16 @@
 [role="xpack"]
 [testenv="basic"]
 [[node-lifecycle-api]]
-== Node lifecycle API
+== Node lifecycle APIs
```

Comment:
Are there other expected/planned lifecycle APIs? Just wondering if we need this level in the hierarchy.